logic/transaction.js

```diff
@@ -1289,6 +1279,14 @@ Transaction.prototype.schema = {
 			type: 'string',
 			format: 'signature',
 		},
+		signatures: {
+			type: 'array',
+			items: {
+				type: 'string',
+				format: 'signature',
+			},
+			uniqueItems: true,
+		},
```

Comment:
@diego-G You added this schema but don't see any relevant change to any other code place.

Reply:
I've updated the branch. This commit was related to the other fix which is already merged.